docs/pull_requests.md

```diff
@@ -0,0 +1,89 @@
+# Pull Request process
+
+Pull Requests in the `openshift-ansible` project follow a
+[Continuous](https://en.wikipedia.org/wiki/Continuous_integration)
+[Integration](https://martinfowler.com/articles/continuousIntegration.html)
+process that is similar to the process observed in other repositories such as
+[`origin`](https://github.com/openshift/origin).
+
+Whenever a
+[Pull Request is opened](../CONTRIBUTING.md#submitting-contributions), some
+automated test jobs must be successfully run before the PR can be merged.
+
+Some of these jobs are automatically triggered, e.g., Travis and Coveralls.
+Other jobs need to be manually triggered by a member of the
+[@team-openshift-ansible-contributors](https://github.com/orgs/openshift/teams/team-openshift-ansible-contributors) team.
+
+## Triggering tests
+
+We have two different Jenkins infrastructures, and, while that holds true, there
+are two commands that trigger a different set of test jobs. We are working on
+simplifying the workflow towards a single infrastructure in the future.
+
+- **Test jobs on the older infrastructure**
+
+  Team members can trigger the set of test jobs in the older infrastructure by
+  writing a comment with the exact text `aos-ci-test` and nothing else.
+  <!--
+    REVIEW: who exactly has permission to trigger this -- define "team" above
+    -- Is it all of team-openshift-ansible-contributors? Or a more general group?
+  -->
+
+  The Jenkins host is not publicly accessible. Test results are posted to S3
+  buckets when complete, and links are available both at the bottom of the Pull
+  Request page and as comments posted by
+  [@openshift-bot](https://github.com/openshift-bot).
+
+- **Test jobs on the newer infrastructure**
+
+  Team members can trigger the set of test jobs in the newer infrastructure by
+  writing a comment containing `[test]` anywhere in the comment body.
```

Comment:
Similar to the question of who can trigger `aos-ci-test`, who can trigger `[test]`? I could try to look it up in code in aos-cd-jobs or the test-pull-requests repo.